<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-68KUQKT" failures="3" tests="4" name="Smoke" time="0.084" errors="0" timestamp="2024-09-30T17:01:07 IST">
  <testcase classname="testng.Skip_Test" name="login" time="0.014">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testng.Skip_Test.login(Skip_Test.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="testng.Skip_Test" name="logOut" time="0.0">
    <skipped/>
  </testcase> <!-- logOut -->
  <testcase classname="testng.SoftAssertDemo" name="test1" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testng.SoftAssertDemo.test1(SoftAssertDemo.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="testng.SoftAssertDemo" name="test2" time="0.004">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at testng.SoftAssertDemo.test2(SoftAssertDemo.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
</testsuite> <!-- Smoke -->
